sherpa/csrc/rnnt_model.h

```diff
@@ -53,7 +53,7 @@ class RnntModel {
    * @param features  A 3-D tensor of shape (N, T, C).
    * @param features_length A 1-D tensor of shape (N,) containing the number of
    *                       valid frames in `features`.
-   * @return Return a tuple containing two tensors:
+   * @return Return a pair containing two tensors:
```

Comment:
I think we will need `feature_lens` in `fast_beam_search`. But surely we might add it when needed.